main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	net_http "net/http"
+	"os"
+	"riot-developer-proxy/handlers/http"
+	"riot-developer-proxy/internal/services"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+)
+
+func init() {
+	if err := godotenv.Load(); err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	hc := &http.HTTPController{
+		RiotClient: initRiotClient(),
+	}
+
+	e := echo.New()
+	e.Use(middleware.Recover())
+
+	e.Any("/*", hc.ProxyToRIOTApi)
+
+	e.Logger.Fatal(e.Start(":" + os.Getenv("APP_PORT")))
```

Comment:
Is this standard practice? It seems a little unclear, but I assume that this could also be a bit clearer:
```
if err := e.Start(...); err != nil {
  e.logger.Fatal(err)
}

Reply:
Yes, is only boilerplate. I'll rewrite for the sample app.

Ps: It's also not production ready, for that we would need to implement `echo.Shutdown()` for a graceful shutdown